Modules/Microsoft365DSC/DSCResources/MSFT_MEMEndpointProtectionApplicationControlPolicyWindows10/readme.md

```diff
@@ -0,0 +1,4 @@
+
+# MEMEndpointProtectionApplicationControlPolicyWindows10
```

Comment:
Update name and description to reflect renaming

Reply:
Updated